languages/el.ini

```
@@ -453,6 +453,7 @@ Exception = "Εξαίρεση"
 Excerpt = "Απόσπασμα"
 exclude_facet = "[εξαιρέσεις]"
 exclude_newspapers = "Χωρίς άρθρα εφημερίδων"
+excluded = "Εξαιρούνται:"
```

```suggestion
excluded = "Εξαιρείται:"
```
Updating translation per request from @sbujam